MemeifyMe/app/src/main/java/accesscode/c4q/nyc/memeifyme/MainActivity.java

```diff
@@ -20,16 +25,33 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         camera_button = (Button) findViewById(R.id.camera_button);
```

Comment:
I think we should stick to a naming convention of general -> specific. So these buttons, for example, would be button_camera, button_album, and so on. I've also seen btn used, particularly in XML.